src/plugins/dashboard/public/application/index.scss

```diff
@@ -4,6 +4,8 @@
 @import './embeddable/panel/index';
 @import './embeddable/viewport/index';
 
+@import '../../../kibana_react/public/exit_full_screen_button/index';
```

Comment:
These relative urls get quite crazy. This is a good opportunity to just import the full screen button styles directly in the JS file.